### PR TITLE
test: Bump CUDA images

### PR DESCRIPTION
**Pull request by ameukam**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:

Bump the base image to a supported Ubuntu version

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by mkumatag** [comment]
But I don't see a presubmit job for building this image.. need a manual build and check if its really builds?
/hold

**Comment by ameukam** [comment]
> But I don't see a presubmit job for building this image.. need a manual build and check if its really builds? /hold

@mkumatag Looks like the s390x architecture is only available on the 11.1 version. The following versions only support amd64 and arm64.

File: test/images/cuda-vector-add/BASEIMAGE [hunk 1]
```diff
@@ -1,3 +1,3 @@
-linux/amd64=nvidia/cuda:11.1.1-devel-ubuntu18.04
-linux/arm64=nvidia/cuda:11.1.1-devel-ubuntu18.04
-linux/ppc64le=nvidia/cuda:11.1.1-devel-ubuntu18.04
+linux/amd64=nvidia/cuda:11.4.3-devel-ubuntu20.04
+linux/arm64=nvidia/cuda:11.4.3-devel-ubuntu20.04
+linux/ppc64le=nvidia/cuda:11.4.3-devel-ubuntu20.04
```
Comment (mkumatag) [inline test/images/cuda-vector-add/BASEIMAGE:3]: Seems like ubuntu flavour is not supported for ppc64le per tags

image

Lets remove ppc64le from the list for now and proceed further.

**Review by mkumatag — CHANGES_REQUESTED** [review]

**Comment by mkumatag** [comment]
But million $$ question is - where is the presubmit jobs for testing changes in `test/images` dir? am I hallucinating all these days? 

/test ?

**Comment by mkumatag** [comment]
> But million $$ question is - where is the presubmit job for testing changes in `test/images` dir? am I hallucinating all these days?

https://github.com/kubernetes/kubernetes/issues/123633 created to track this

File: test/images/cuda-vector-add/BASEIMAGE (hunk 1 above)
Comment (ameukam) [reply test/images/cuda-vector-add/BASEIMAGE]: Done.

**Comment by mkumatag** [comment]
Image promotion PR - https://github.com/kubernetes/k8s.io/pull/6512

**Pull request merged**